pkg/provider/azure_backoff.go

```diff
@@ -385,25 +385,6 @@ func (az *Cloud) ListLB(service *v1.Service) ([]network.LoadBalancer, error) {
 	return allLBs, nil
 }
 
-// ListPIP list the PIP resources in the given resource group
```

Comment:
Since here the UT of old `ListPIP` is removed, can we have a new UT for the new `listPIP` in `azure_wrap_test.go`?

Reply:
yes, will do. Thanks